.github/workflows/main.yml

```diff
@@ -9,8 +9,10 @@ on:
     branches: [ main ]
 env:
   IMAGE: sample
-  REGISTRY_HOSTNAME: eu.gcr.io
-  HOST: ${{ secrets.GOOGLE_PROJECT_ID }}
+  REGISTRY_HOSTNAME: europe-west2-docker.pkg.dev
+  SPINNAKER_GOOGLE_PROJECT_ID: ${{ secrets.SPINNAKER_GOOGLE_PROJECT_ID }}
+  GAR_REPOSITORY: images
+  GAR_GOOGLE_PROJECT_ID: ${{ secrets.GAR_GOOGLE_PROJECT_ID }}
   RELEASE_HOST: ${{ secrets.RELEASE_PROJECT_ID }}
```

Comment:
Is `RELEASE_HOST` now redundant? If so this should be removed.

Reply:
Yes, it's redundant. I've removed it now.